src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/StringValidators.java

```diff
@@ -75,4 +75,11 @@ private static boolean isNotEmpty(String toTest, String qualifiedFieldName, Erro
         }
         return true;
     }
+
+    public static void checkIsEmpty(String toTest, String qualifiedFieldName, Errors errs, String loggingContext) {
+        if (toTest != null && !toTest.trim().isEmpty()) {
+            setErrorMsgToLocation(errs, qualifiedFieldName, ValidationMessages.SHOULD_NOT_BE_POPULATED_ERROR_MESSAGE.replace("%s", qualifiedFieldName));
```

Comment:
I would suggest using the String 'format' method here, rather than 'replace'. Examples already elsewhere.